app/controllers/reservations_controller.rb

```diff
@@ -1,6 +1,10 @@
 class ReservationsController < ApplicationController
```

Comment:
are you going to authorize the access to this data? It's quite sensitive

Reply:
I will, as soon as I manage user and roles